### PR TITLE
New superstamp HLSP blog [WIP]

### PR DESCRIPTION
**Pull request by amcody**

### CODE REVIEW

File: content/news/2017-clusterworkshop-schedule.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Comment (mirca) [inline content/news/2017-clusterworkshop-schedule.md]: There is some debris possibly from a ``git pull`` or ``git rebase``

File: content/news/2017-clusterworkshop-schedule.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+to take place from January 16 to 18, 2018, on the BU campus in Boston, Massachusetts. The first two days consist
+of scientific talks by attendees, while the last day will be an "unconference" dedicated to tutorials on K2 data 
+use as well as participant collaboration on new ideas for data management and software tools.
+
+Talk submission is now closed and a preliminary schedule has been posted on the [workshop website](/cluster-workshop).
+
+<img class="img-responsive" style="max-width:100%;" src="images/k2/bu_fireworks.jpg" alt="Dwarf Stars and Clusters with 
+K2: A Workshop"> 
+*Image Credit: Eric Kilby*
+=======
```
Comment (mirca) [inline content/news/2017-clusterworkshop-schedule.md]: same here

File: content/news/2017-clusterworkshop-schedule.md [hunk 3]
```diff
@@ -16,3 +32,4 @@ tutorials and collaborations.
 We continue to accept late registrations for additional participants.
 A small number of talk slots remain available on a first-come, first-served basis.
 Please contact <a href="mailto:keplergo@mail.arc.nasa.gov">keplergo@mail.arc.nasa.gov</a> as soon as possible to be considered for the spare talk slots.
+>>>>>>> a978ade714e9a200c07ff7479202a6cff5590671
```
Comment (mirca) [inline content/news/2017-clusterworkshop-schedule.md]: and here

**Comment by barentsen** [comment]
@amcody Can you re-do this PR by branching from a clean version of master that is in sync with KeplerGO's master?

**Comment by mirca** [comment]
@amcody I can help with that if needed!

**Pull request closed without merging**